apertium-tur-uzb.tur-uzb.dix

```diff
@@ -4323,11 +4357,13 @@
        <e><p><l>da<s n="adv"/></l><r>da<s n="adv"/></r></p></e>
        <e><p><l>olsa<b/>da<s n="adv"/></l><r>boʻlsada<s n="adv"/></r></p></e>
        <e><p><l>fikrine<b/>göre<s n="adv"/></l><r>fikricha<s n="adv"/></r></p></e>
-       <e><p><l>kaç<s n="adv"/></l><r>nechta<s n="adv"/></r></p></e>
+       <e><p><l>kaç<s n="det"/><s n="qnt"/></l><r>nechta<s n="det"/><s n="qnt"/></r></p></e>
        <e><p><l>neredeyse<s n="adv"/></l><r>qariyb<s n="adv"/></r></p></e>
        <e><p><l>daim<s n="adv"/></l><r>doim<s n="adv"/></r></p></e>
+       <e><p><l>her<b/>daim<s n="adv"/></l><r>har<b/>doim<s n="adv"/></r></p></e>
+       <e><p><l>her<b/>zaman<s n="adv"/></l><r>har<b/>doim<s n="adv"/></r></p></e>
        <e><p><l>avlan<s n="v"/><s n="iv"/></l><r>ovlan<s n="v"/><s n="iv"/></r></p></e>
-       <e><p><l>çalış<s n="v"/><s n="iv"/></l><r>ishlay<s n="v"/><s n="iv"/></r></p></e>
+       <e><p><l>çalış<s n="v"/><s n="iv"/></l><r>ishla<s n="v"/><s n="iv"/></r></p></e>
```

Comment:
"çalış" is transitive

Reply:
Could you give an example?

Reply:
bu kitabı çalış

Reply:
That doesn't seem to translate as ishla..  What does it mean in this context?